src/Simulation/Native/Makefile

```diff
@@ -0,0 +1,364 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Makefiles are generated by CMake... should be checked in

Reply:
Is this still a concern? The file is present in the change, but perhaps you comment meant to say "shouldn't" be checked in? I could remove that file if that is preferable.

Reply:
Yeah... shouldn't (sorry)